docs/data.rst

```diff
@@ -100,12 +100,25 @@ datasets.
 These datasets are directly accessible from the Google Cloud Pangeo deployment:
 `pangeo.pydata.org <http://pangeo.pydata.org>`_.
 
-+--------+--------------+
-| Name   | Description  |
-+========+==============+
-| Under  | Construction |
-+--------+--------------+
-
+.. raw:: html
+
+    <table class="table table-striped large">
+    {% for name in catalog.walk() %}
+      {% set entry = catalog[name] %}
+
+      <tr>
+        <td>
+          <ul class="list-inline">
+            <li><strong>{{ name }}</strong></li><br>
+            <li>{{ entry.description }}</li><br>
+            <li><em>Format: {{ entry.describe_open()['plugin'] }}</em></li>
+            <li><em>Container: {{ entry.container }}</em></li><br>
+            <li><em>Location: {{ entry.urlpath }}</em></li>
+          </ul>
+        </td>
+      </tr>
+    {% endfor %}
+    </table>
```

Comment:
Today I learned that you can include templated HTML in sphinx docs :)

Reply:
It might be useful to provide a code snippet that people can copy-paste to get a dataset within a Jupyter notebook

```python
import intake
ds = intake....('gcs://...')
```

cc @martindurant

Reply:
Agreed. Without any installation of conda data packages or preloading, the syntax would be like

```
s = intake.open_catalog('gcs://...').mydata.to_dask()
```

Reply:
Perhaps we should consider packaging / preloading our catalog by default on the pangeo clusters? This [blog post](https://www.informaticslab.co.uk/home/2018/9/13/tutorial-how-to-build-an-intake-catalog) by @jacobtomlinson explains one approach.